examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -519,7 +519,7 @@ export const TaskListInstantiationFactory = new DataObjectFactory<TaskList>(
  */
 export class BaseDocument extends DataObject implements IBaseDocument {
 	private readonly taskListCollection = new Map<string, TaskList>();
-
+	private readonly leader = "leader";
```

Comment:
Nit, to be super clear that this isn't the actual value of the leader id.
```suggestion
	private readonly leaderKey = "leader";
```